Resources/Prototypes/_Goobstation/Polymorphs/polymorph.yml

```
@@ -42,3 +42,14 @@
     revertOnDeath: true
     transferName: true
     revertOnCrit: true
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)